bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -43,52 +53,14 @@ class WheelZoomToolView extends GestureTool.View
 
     factor  = @mget('speed') * delta
```

Comment:
@bryevdv, thanks for the tip.  I'll be on the lookout for that in the files I touch.